package.json

```diff
@@ -46,8 +46,10 @@
     "@fortawesome/react-fontawesome": "^0.1.18",
     "@popperjs/core": "^2.11.4",
     "airbnb-prop-types": "^2.16.0",
+    "axios": "^0.27.2",
     "bootstrap": "^4.6.1",
     "classnames": "^2.3.1",
+    "dotenv": "^16.0.1",
```

Comment:
[curious] Why were these packages added to the root package.json file?

Reply:
@Jawayria Bump.

Reply:
I guess there was a mistake, thanks for pointing out.